arangod/Aql/AqlItemBlockInputMatrix.cpp

```diff
@@ -64,26 +57,20 @@ AqlItemBlockInputRange& AqlItemBlockInputMatrix::getInputRange() {
   if (_lastRange.hasDataRow()) {
     return _lastRange;
   }
-  // Need initialze lastRange
+  // Need initialize lastRange
   if (_aqlItemMatrix->numberOfBlocks() == 0) {
     _lastRange = {AqlItemBlockInputRange{upstreamState()}};
   } else {
-    auto [blockPtr, start] =  _aqlItemMatrix->getBlock(_currentBlockRowIndex);
+    auto [blockPtr, start] = _aqlItemMatrix->getBlock(_currentBlockRowIndex);
     ExecutorState state = incrBlockIndex();
     _lastRange = {state, 0, std::move(blockPtr), start};
   }
   return _lastRange;
 }
 
-SharedAqlItemBlockPtr AqlItemBlockInputMatrix::getBlock() const noexcept {
-  TRI_ASSERT(_aqlItemMatrix == nullptr);
-  return _block;
-}
-
-std::pair<ExecutorState, AqlItemMatrix const*> AqlItemBlockInputMatrix::getMatrix() noexcept {
+std::pair<ExecutorState, AqlItemMatrix const*>
+AqlItemBlockInputMatrix::getMatrix() noexcept {
   TRI_ASSERT(_aqlItemMatrix != nullptr);
-  TRI_ASSERT(_block == nullptr);
-  TRI_ASSERT(!_shadowRow.isInitialized());
```

Comment:
Why did you remove these assertions?

Reply:
Ups. "By accident". This might happen during the cherry-pick. Will revert this.

Reply:
re-added asserts.